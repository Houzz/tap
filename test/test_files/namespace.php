<?php

namespace test\space {

function testF() {
}

class A {
    const C = "str";
}

}

namespace test\space2 {

function testF() {  // not a dupliation
    $a = A::C;   // A::C not in namespace test\space2;
    __tap_check(\test\space\A::C, "string");
}

}

namespace test {
    function testF() {
        $a = space\A::C;
        __tap_check($a, "string");
    }
}
