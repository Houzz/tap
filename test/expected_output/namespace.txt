17 E_CONST_NOT_RESOLVED
